docker/prod/Jenkinsfile

```diff
@@ -129,6 +129,10 @@ EOL""")
             sh "kubectl --context ${k8sContext} -n ${environment} apply -f app/docker/prod/k8s.yaml"
             rolloutStatus = sh(returnStatus: true, script: "kubectl --context ${k8sContext} -n ${environment} rollout status deployment/mediawiki-'${environment}'")
         }
+
+        withDockerContainer(kubectlImage) {
```

Comment:
does this create new docker instance?
maybe this could be run in previous withDockerContainer?